src/platform/driver_init.c

```diff
@@ -115,6 +121,57 @@ static void _spi_init(void)
     SPI_OLED_enable();
 }
 
+static void _spi_mem_CLOCK_init(void)
```

Comment:
why uppercase clock and not lowercase? (in our own code we usually have lowercase snake case, and in this file there is also a `_ptc_clock_init()` function named similarly).

Reply:
Right, it was a leftover of the original Atmel start function. Fixed :+1: